engine.py

```diff
@@ -23,200 +19,240 @@
 import nukescripts
 
 class NukeEngine(tank.platform.Engine):
+    """
+    An engine that supports Nuke 6.3v5+ and Hiero 9.0v*+.
+    """
+    # Define the different areas where menu events can occur in Hiero.
+    (HIERO_BIN_AREA, HIERO_SPREADSHEET_AREA, HIERO_TIMELINE_AREA) = range(3)
 
-    ##########################################################################################
-    # init
+    def __init__(self, *args, **kwargs):
+        self._hiero = nuke.env.get("hiero")
```

Comment:
Definitely agreed! I'm also going to rename has_ui to be ui_enabled to match. In the near future we will then also have a studio_enabled property to go along with them.